pip_env/requirements.txt

```diff
@@ -0,0 +1,10 @@
+dask==2022.7.1
+torch==2.2.0
+torchvision==0.17.0
+torchaudio==2.2.0
+scikit-learn==1.3.1
+xarray==2024.2.0
+h5py==3.10.0
+zarr==2.17.0
+rechunker==0.5.2
+
```

Comment:
Pinning each dependency at a fixed version is not appropriate for a library. Users are forced to up- and downgrade packages their own environment which is often not desired or even not possible.

Reply:
See also https://stackoverflow.com/a/44938662

If you really need a pinned environment for a dedicated application that `mltools` is used in, you can consider a Docker image or follow suggestions given in https://pip.pypa.io/en/stable/topics/repeatable-installs/

Reply:
requirements are updated in the [pyproject.toml](https://github.com/deepesdl/ML-Toolkits/blob/develop/pyproject.toml)